core/templates/pages/email-dashboard-pages/email-dashboard-page.component.spec.ts

```diff
@@ -133,7 +133,7 @@ describe('Email Dashboard Page', function() {
     expect(ctrl.editedFewerThanNExps).toBe(null);
   });
 
-  it('should get next page of queries', function() {
+  it('should get next page of queries when going to next page', function() {
```

Comment:
for this test and the next, I think you can remove the "when ...." part as it is repetitive

Reply:
Done.